recipe-finder-fe/src/Pages/NotFound/index.tsx

```diff
@@ -0,0 +1,23 @@
+import Header from "../../Components/Header";
+
+function NotFound() {
+  return (
+    <>
+      <Header title="FoodHub" />
```

Comment:
Using the Header again inside your page components causes a 'double header' throughout the app which just looks weird.

You need to distinguish between a page header and an app hero/title component I think?